content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -32,6 +32,12 @@ The `env` and `envFrom` fields have different effects.
  are set as environment variables for the container.
  You can also specify a common prefix string.
 
+{{< note >}}
+The allowed character range of env name is within C_IDENTIFIER,
```

Comment:
I think a lot of readers won't know what `C_IDENTIFIER` means and could be confused by this statement.

Reply:
I followed our comments in the api documentation.
ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/

Maybe they'll be confused, until they use a search engine....

Reply:
If our docs site doesn't explain what a `C_IDENTIFIER` means (it doesn't really), then we shouldn't refer to it.

Instead, we could add a small page about environment variables for containers (inside https://kubernetes.io/docs/concepts/configuration/) - just a stub for now, and then add a heading about allowed characters inside variable names.

Reply:
all right.